### DISCUSSION TITLE
Neden İbax

### DISCUSSION THREAD

**Discussion by Omrumbenim** [IBAX-io/go-ibax#1723] [Bounty Hunter]
Son saatler henüz kaçırmış değilsiniz, katılın  ve görevleri tamamlayın

**Comment by Omrumbenim** [comment]
😄

**Comment by Abdullahoda** [comment]
bizim için bir fırsat stake ve madencılık olucak nasıl yapıcaklar merak ediyoruz umarım ekran kartı gerekmez yorumu cevaplandı olarak işaretlersen sevinirim..
